src/main/java/org/folio/gobi/GobiPurchaseOrderParser.java

```diff
@@ -113,34 +111,42 @@ public Input(String publicId, String sysId, InputStream input) {
       this.inputStream = input;
     }
 
+    @Override
```

Comment:
I wonder how all of these were working before without `@Override`...

Reply:
From:  https://docs.oracle.com/javase/tutorial/java/annotations/predefined.html

> @Override @Override annotation informs the compiler that the element is meant to override an element declared in a superclass. Overriding methods will be discussed in Interfaces and Inheritance.
> 
> ```
>   // mark method as a superclass method
>   // that has been overridden
>   @Override 
>   int overriddenMethod() { }
>```
> While it is not required to use this annotation when overriding a method, it helps to prevent errors. If a method marked with @Override fails to correctly override a method in one of its superclasses, the compiler generates an error.